src/AMQPEventsSender.js

```diff
@@ -18,26 +18,33 @@ class AMQPEventsSender extends EventEmitter {
   /**
    * @constructor
    * @param {amqplib.Connection} amqpConnection
-   * @param {String} queueName endpoint address, to send data to
+   * @param {Object} params
+   * @param {String} [params.queueName=''] queue for feedback from AMQPRPCServer,
+   *    default is '' which means auto-generated queue name, should correspond with AMQPEventsSender
+   * @param {Number} [params.TTL=AMQPEventsSender.TTL] TTL of messages
    */
-  constructor(amqpConnection, queueName) {
+  constructor(amqpConnection, params = {}) {
```

Comment:
Seems that autogenerated queue name is sensless here. Reasons:
1) we have no queue name to send to. And we are not creating it. Sequentially we'll never succeed to send anything to anywhere
Options: 
1) create queue here, and make possible to get it's name, to make possible to send it to someone to make him listen
2) disallow to work with anonymous queues here => let someone create this queue for us before we begin to work